spec/gem_update_spec.rb

```diff
@@ -10,12 +10,14 @@
           github_token: 'test',
           update_limit: nil,
           repositories: 'schasse/outdated',
-          projects: 'outdated:project_folder')
+          projects: 'schasse/outdated:project_folder')
```

Comment:
the specs were never actually working for the project folder because of a hash key mismatch in `update_gems.rb#276/277`.

Now that they do, I get the following error on that spec:
```
COMMAND FAILED: git checkout master Gemfile.lock
...
COMMAND FAILED: GIT_MERGE_AUTOEDIT=no git pull origin master
     #   ./update_gems.rb:41:in `run'
```

Any thoughts on that @schasse @gjhernandez ?

Reply:
`pull origin master` actually does a merge, probably into `update_schlib` branch. In travis we don't have any more debug output unfortunately...

Reply:
the problem was, that the change to the project dir was before the master merge where the project dir did not exist yet.